### PR TITLE
making "Dual RP/SUP" to be required for the test to pass

### PR DESCRIPTION
**Pull request by averasko**
PiperOrigin-RevId: 494593179

making "Dual RP/SUP" to be required for the test to pass (Arista 7208 and Nokia will start failing it)

### CODE REVIEW

**Review by sezhang2 — DISMISSED** [review]

**Review by liulk — APPROVED** [review]

File: internal/fptest/doc.go [hunk 1]
```diff
@@ -14,3 +14,9 @@
 
 // Package fptest provides helper functions for organizing tests and reporting results.
 package fptest
+
+// Import packages for test flags so that the presence of unused flags won't break tests.
+import (
+	_ "google3/third_party/openconfig/featureprofiles/internal/args/args"             // keep
```
Comment (liulk) [inline internal/fptest/doc.go:20]: Please fix the import paths.

**Review by liulk — CHANGES_REQUESTED** [review]

File: internal/fptest/doc.go [hunk 2]
```diff
@@ -14,3 +14,9 @@
...
+	_ "google3/third_party/openconfig/featureprofiles/internal/deviations/deviations" // keep
```
Comment (liulk) [inline internal/fptest/doc.go:21]:
```suggestion
	_ "github.com/openconfig/featureprofiles/internal/args/args"             // keep
	_ "github.com/openconfig/featureprofiles/internal/deviations/deviations" // keep
```

**Review by sezhang2 — APPROVED** [review]

File: internal/fptest/doc.go [hunk 3]
```diff
@@ -14,3 +14,9 @@
 
 // Package fptest provides helper functions for organizing tests and reporting results.
 package fptest
+
+// Import packages for test flags so that the presence of unused flags won't break tests.
+import (
+	_ "github.com/openconfig/featureprofiles/internal/args/args"             // keep
+	_ "github.com/openconfig/featureprofiles/internal/deviations/deviations" // keep
```
Comment (liulk) [inline internal/fptest/doc.go:21]:
```suggestion
	_ "github.com/openconfig/featureprofiles/internal/args"             // keep
	_ "github.com/openconfig/featureprofiles/internal/deviations" // keep
```

**Review by sezhang2 — APPROVED** [review]

**Review by sezhang2 — APPROVED** [review]

**Review by liulk — APPROVED** [review]

**Pull request merged**